### ISSUE TITLE
Feature not working on newer chromedrivers

### ISSUE THREAD

**Issue by sbhaaf** [SeleniumHQ/selenium#6754]
Hello all,

I'm having selective trouble getting an infinite scroll event to trigger. 

All of my devices are Ubuntu 16.04, and  I'm using the Python API. On my local node I use chrome-driver 2.25 and was able to get the feature to run correctly. On a remote node with chrome-driver 2.37 the event will not trigger. I updated the remote node to 2.45 and it still did not trigger, then I updated the local node to 2.45 and now it won't trigger either. What I believe is that there was an update somewhere after 2.25 that made it impossible for this event to function correctly.

The way this infinite page works is when you scroll to the bottom of the page then the application displays a full-page loader, then renders new data, then removes the full-page loader and is then ready to be triggered again.

What worked in 2.25 and the JavaScript console was to wait until the loader disappeared and then use:

```
driver.execute_script('scrollTo(0, 0)')
driver.execute_script('scrollTo(0, 1000000)')
```

This same code using chrome-driver 2.37 or 2.45 does not trigger the event. I've tried other methods, including:
* Using using the **Visual Event** chrome extension to track down the element that has the scroll event, and then manually triggering with:
  - `element.dispatchEvent(new Event('scroll'))`
    + works in console, did not work with 2.25, 2.37 or 2.45.
* Jumping to the` <footer>` of the page, both with `ActionChains` and `driver.exeute_script`:
  - `ActionChains(driver).moveToElement(driver.find_element_by_css_selector('footer'))`
  - `driver.execute_script("document.querySelector('footer').scrollIntoView()")`
    + works in console, did not work with 2.45 and did not test in 2.25 or 2.37.
* Resizing the window to (500, 5000)
  - Does not trigger the scroll event in 2.45 and did not test in 2.25 or 2.37.

It's interesting because the page will run the scroll event on initial load with any chrome-driver, but nothing I do in 2.37 or 2.45 will get it to fire after that. 

**TL;DR:** The page's scroll event will trigger with chrome-driver 2.25, but not 2.37 or 2.45. 

Any help or thoughts here would be greatly appreciated.

**Comment by cgoldberg** [comment]
please submit chromedriver issues with the chromium project: http://code.google.com/p/chromedriver/issues/
